enctests/testframework/encoders/ffmpeg_encoder.py

```diff
@@ -178,9 +179,11 @@ def get_output_filename(self, test_name: str) -> pathlib.Path:
         if stem[-1] == ".":
             stem = stem[:-1]
     
+        testbasename = f"{stem}-{test_name}"
+
         out_file = self.destination.absolute().joinpath(
-            f"{stem}-{test_name}{self.test_config.get('suffix')}"
+            f"{testbasename}{self.test_config.get('suffix')}"
         )
 
-        return out_file
+        return (out_file, testbasename)
```

Comment:
We need to update the return type of this function. `-> Tuple[pathlib.Path, str]`

```suggestion
        return out_file, testbasename
```
Python will return this as a tuple without the parentheses.

Reply:
Should be fixed in the most recent patch.